lib-multisrc/flixscans/src/eu/kanade/tachiyomi/multisrc/flixscans/FlixScans.kt

```diff
@@ -169,52 +162,56 @@ abstract class FlixScans(
 
     override fun searchMangaRequest(page: Int, query: String, filters: FilterList): Request {
         if (query.isNotEmpty()) {
-            return postPath("search/serie/${query.trim()}?page=$page")
+            return GET("$apiUrl/search/serie/${query.trim()}?page=$page")
```

Comment:
I don't think this is sufficient in terms of encoding `query`
`headers` is also missing